master/jenkins_files/files/home/jenkins-slave/cleanup_docker_images.py

```diff
@@ -148,16 +131,18 @@ def run_container_cleanup(args, minimum_age):
     parser.add_argument('--logfile', type=str,
                         default='/var/log/jenkins-slave/cleanup_docker_images.log',
                         help='Where to log output')
-    parser.add_argument('--min-days', type=int, default=1,
+    parser.add_argument('--min-days', type=int, default=0,
                         help='The minimum age of items to clean up in days.')
-    parser.add_argument('--min-hours', type=int, default=0,
+    parser.add_argument('--min-hours', type=int, default=6,
```

Comment:
Please change the min-hours to `8` since 480 minutes is the timeout for arm jobs.